wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp

```diff
@@ -8,7 +8,7 @@
 
 #include "HumanDynamicsWrapper.h"
 #include "IHumanDynamics.h"
-#include "thrift/HumanDynamics.h"
+#include <HumanDynamicsEstimation/HumanDynamics.h>
```

Comment:
@kouroshD maybe something related to this?

Reply:
here is related to HDE repo, but the error is in wearables, which has dependency on that.

Reply:
I mean, something similar, related to the change of the include path. As @lrapetti wrote, it might be related to the issue he linked.